tests/torch/test_models/synthetic.py

```diff
@@ -501,3 +501,24 @@ def forward(self, x):
         unbinded_processed[0] = self.conv4(y_unbinded[0])
         y = torch.cat(unbinded_processed, axis=0)
         return y
+
+
+class ModelExtractionModel(torch.nn.Module):
```

Comment:
Name of classes does not reflect the modules.
And looks like it should be moved to `tests/torch/fx/...`

Reply:
This is an instance of `torch.nn.Module`, I see no contradictions here. There are several models named after a test which is using this model (ModelForGraphBuildingTest and etc). I would keep it here until it is necessary

Reply:
Done